src/main/asciidoc/analytics/twitter-analytics/main.adoc

```diff
@@ -37,12 +46,12 @@ Created new stream 'tweets'
 ```
 +
 ```
-dataflow:>stream create tweetlang  --definition ":tweets.twitterstream > field-value-counter --fieldName=lang --name=language" --deploy
+dataflow:>stream create tweetlang  --definition ":tweets.twitterstream > counter --name=language --counter.tag.expression.lang=#jsonPath(payload,'$..lang')" --deploy
```

Comment:
Since there's a behavior change, perhaps it'd be good to show a sample of a Tweet JSON and explain why we are using `payload,'$..lang'` to pull the language text from the JSON. This used to be hidden from the end-users previously.